interface-elements-for-web/articles/report-designer/create-reports/master-detail-reports-with-detail-report-bands.md

```diff
@@ -16,7 +16,7 @@ This tutorial illustrates how to display hierarchical data in a master-detail re
 
```

Comment:
Can you make the following changes to the lines above?

Line 7:
`This approach is effective if your data source contains master-detail relationship.`

`This technique is effective if your data source contains a master-detail relationship.`

Line 13:
`2. [Bind the report](../bind-to-data.md) to a required data source and provide it with a master-detail relationship as described in the [Bind a Report to a Database](../bind-to-data/bind-a-report-to-a-database.md) topic.`

a) Please change "a required data source" to "the required data source".
b) Update this sentence without using "provide".

Reply:
fixed